lib/core/imports/index.js

```diff
@@ -40,10 +40,36 @@ if (window.Uint32Array) {
   }
 }
 
+/**
+ * @deprecated
+ */
+class CSSSelectorParser {
```

Comment:
This class isn't a 1:1 back compatible wrapper. The return of `CssSelectorParser` was a class, but the class also had additional functions that were used to configure the parser (`registerSelectorPseudos`, `registerNestingOperators`, etc.). In order to fully be back compatible, this class would need to implement those functions and probably then call `createParser` in the `parse` function with all the settings the user has set using the configuration functions.